beetsplug/convert.py

```diff
@@ -208,7 +208,7 @@ def encode(self, command, source, dest, pretend=False):
         except OSError as exc:
             raise ui.UserError(
                 u"convert: could invoke '{0}': {1}".format(
-                    ' '.join(args), exc
+                    ' '.join(ui.decargs(args)), exc
```

Comment:
Also, if we're now going to be joining Unicode strings, the space should probably be a Unicode literal too: `u' '`.